### PR TITLE
Merging develop into 1.2.0-rc2

### PR DESCRIPTION
**Pull request by manojsp12**

### CODE REVIEW

File: id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/builder/IdentityIssuanceProfileBuilder.java [hunk 1]
```diff
@@ -45,36 +52,54 @@
...
-		buildNewProfile();
-		IdentityIssuanceProfile profile = new IdentityIssuanceProfile();
-		profile.setProcessName(processName);
-		profile.setDate(LocalDate.now());
-		profile.setOldProfile(oldProfile);
-		profile.setNewProfile(newProfile);
-		return profile;
+		try {
+			if (StringUtils.isBlank(filterLanguage))
+				IdentityIssuanceProfileBuilder.setFilterLanguage(this.getPreferredLanguage(newIdentity));
```
Comment (LoganathanSekar7627) [inline id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/builder/IdentityIssuanceProfileBuilder.java:57]: Setting preferred language statically may not be appropriate for a multiple parellel request processing.

File: id-repository/credential-service/src/main/java/io/mosip/credentialstore/provider/CredentialProvider.java [hunk 2]
```diff
@@ -346,24 +333,30 @@ public DataProviderResponse getFormattedCredentialData(
...
+		try {
+			String subType;
+			if (bdbSubTypeList.size() == 1) {
+				subType = bdbSubTypeList.get(0);
+			} else {
+				subType = bdbSubTypeList.get(0) + " " + bdbSubTypeList.get(1);
+			}
+			return subType;
+		} catch (Exception e) {
+			return null;
```
Comment (LoganathanSekar7627) [inline id-repository/credential-service/src/main/java/io/mosip/credentialstore/provider/CredentialProvider.java:346]: Catch a specific exception. Add logger to the exception, if needed in debug level.

**Review by LoganathanSekar7627 — CHANGES_REQUESTED** [review]

File: id-repository/credential-service/src/main/java/io/mosip/credentialstore/provider/CredentialProvider.java (hunk 2 above)
Comment (manojsp12) [reply id-repository/credential-service/src/main/java/io/mosip/credentialstore/provider/CredentialProvider.java:349]: Fixed

File: id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/builder/IdentityIssuanceProfileBuilder.java (hunk 1 above)
Comment (manojsp12) [reply id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/builder/IdentityIssuanceProfileBuilder.java]: Updated to class variable

**Review by LoganathanSekar7627 — APPROVED** [review]

**Pull request merged**